### PR TITLE
Update Bundles

### PR DESCRIPTION
**Pull request by shayancanonical**
Update bundles with new revisions from Charmhub.

### CODE REVIEW

**Review by paulomach — APPROVED** [review]

**Review by paulomach — APPROVED** [review]

File: releases/latest/mysql-k8s-bundle.yaml [hunk 1]
```diff
@@ -21,14 +16,19 @@ applications:
     channel: latest/edge
     charm: tls-certificates-operator
     constraints: arch=amd64
-    revision: 15
-    scale: 1
```
Comment (taurus-forever) [inline releases/latest/mysql-k8s-bundle.yaml:25]: No more pinning for tls-certificates-operator?

**Review by taurus-forever — APPROVED** [review]
LGTM, but a question to be sure pinning has been removed on purpose.

File: releases/latest/mysql-k8s-bundle.yaml (hunk 1 above)
Comment (shayancanonical) [reply releases/latest/mysql-k8s-bundle.yaml:25]: odd. we should be pinning the tls-certificates-operator. will investigate the script that generates the revisions and trace the reason why the pinning is skipped

**Comment by taurus-forever** [comment]
BTW, failing Lint tests is due to the bug in flake8: https://github.com/savoirfairelinux/flake8-copyright/issues/19
We should pin it to old version 5.0.4 or stop using flake8 (Jon point).

File: releases/latest/mysql-k8s-bundle.yaml (hunk 1 above)
Comment (taurus-forever) [reply releases/latest/mysql-k8s-bundle.yaml:25]: Ignore. I was blind. It was resorted (see line 22-23).

**Review by taurus-forever — APPROVED** [review]

**Review by paulomach — COMMENTED** [review]
lgtm, why the changes in keys orders?

**Pull request merged**